querybook/server/lib/query_analysis/templating.py

```diff
@@ -262,10 +316,10 @@ def render_templated_query(
     jinja_env = get_templated_query_env(engine_id)
     try:
         escaped_query = _escape_sql_comments(query)
-        variables_in_query = get_templated_variables_in_string(escaped_query, jinja_env)
+        variables_in_query = get_templated_variables_in_string(escaped_query)
 
         all_variables = get_templated_query_variables(variables, jinja_env)
-        verify_all_variables_are_defined(variables_in_query, all_variables)
+        verify_all_variables_are_defined(variables_in_query, all_variables, jinja_env)
```

Comment:
if we keep variables_in_query = get_templated_variables_in_string(escaped_query, jinja_env) then verify_all_variables_are_defined won't need to check for global funcs